src/backend/mpi/swdsm.cpp

```diff
@@ -978,10 +970,10 @@ void argo_initialize(std::size_t argo_size, std::size_t cache_size){
 		memset(global_offsets_tbl, 0, offsets_tbl_size_bytes);
 	}
 
-	for(j=0; j<cachesize; j++){
-		cacheControl[j].tag = GLOBAL_NULL;
-		cacheControl[j].state = INVALID;
-		cacheControl[j].dirty = CLEAN;
+	for(std::size_t i=0; i<cachesize; i++){
```

Comment:
Add spaces around `=` and `<`, for consistency with other `for` loops in this file.

Reply:
I've pushed another commit making sure all (hopefully) `for` loops use this standard instead of just fixing this one occurrence.